mission_control/navigator_launch/launch/gnc.launch

```diff
@@ -24,20 +24,9 @@
         <param name="torque_scale" value="500" type="double"/>
     </node>
 
+    <include file="$(find navigator_launch)/launch/subsystems/lqrrt_node.launch"/>
```

Comment:
@mattlangford, I think it is important for either lqRRT or C3 (I would really prefer lqRRT) to come up with the system in the sim. How would you feel about a lake_day argument that skips launching it?